tests/detect-flow-pkts-either/test.rules

```diff
@@ -0,0 +1 @@
+alert ip any any -> any any (msg:"Flow has more than 3000 bytes and 10 pkts in either direction";flow.pkts:either,=10;flow.bytes:either,>3000; sid:1;)
```

Comment:
I guess it would be nice to have the signatures with other directions and see which one match, and which ones do not match

Reply:
Indeed. Will add them. Thank you! 🙇🏽‍♀️